_data-prepper/managing-data-prepper/source-coordination.md

```diff
@@ -0,0 +1,158 @@
+
+If `ttl` is not configured on the table, any items no longer needed in the table must be cleaned manually.
+
+The following shows the full set of permissions needed for Data Prepper to create the table, enable `ttl`, and interact with the table:
+
+```json
+{
+  "Sid": "ReadWriteSourceCoordinationDynamoStore",
+  "Effect": "Allow",
+  "Action": [
+    "dynamodb:DescribeTimeToLive",
+    "dynamodb:UpdateTimeToLive",
+    "dynamodb:DescribeTable",
+    "dynamodb:CreateTable",
+    "dynamodb:GetItem",
+    "dynamodb:PutItem",
+    "dynamodb:Query"
+  ],
+  "Resource": [
+    "arn:aws:dynamodb:${REGION}:${AWS_ACCOUNT_ID}:table/${TABLE_NAME}",
+    "arn:aws:dynamodb:${REGION}:${AWS_ACCOUNT_ID}:table/${TABLE_NAME}/index/source-status"
+  ]
+}
+```
+
+
+| Value | Required | Type | Description |
+| :--- | :--- | :--- | :--- | 
+| `table_name` | Yes | String  | The name of the table to be used for source coordination. |
+| `region` | Yes | String | The region of the table. |
+| `sts_role_arn` | No  | String  |  The `sts` role that contains permissions to the table. Uses default credentials when not provided. |
+| `sts_external_id` | No | String  | The external ID used in the API call to assume the `sts_role_arn`. |
+| `skip_table_creation` | No | Boolean  | If set to `true` when using an existing store, and the attempt to create the store will skip. Default is `false`. |
+| `provisioned_write_capacity_units` | No | Integer |  The number of write capacity units to configure on the table. Default is `10`. |
+| `provisioned_read_capacity_units`  | No | Integer | The number of read capacity units to configure on the table. Default is `10`. |
+| `ttl` | Duration | Optional. The duration of the time to live for the items in the table. This TTL is extended by this duration when an update is made to the item. Defaults to no TTL being used on the table. |
+  
+The following example shows a `dynamodb` store:
+
+```yaml
+source_coordination:
+  store:
+     dynamodb:
+       table_name: "DataPrepperSourceCoordinationStore"
+       region: "us-east-1"
+       sts_role_arn: "arn:aws:iam::##########:role/SourceCoordinationTableRole"
+       ttl: "P7D"
+       skip_table_creation: true
+```
+
+#### In-memory store (default)
+
+The following example shows an `in_memory` store, best used with single node cluster:
+
+
+```yaml
+source_coordination:
+  store:
+    in_memory:
+```
+
+
+## Metrics
+
+Metrics in source coordination are interpreted differently depending on which source is configured. The format of source coordination all metrics is `<sub-pipeline-name>_source_coordinator_<metric-name>`. You can use the sub-pipeline name is used to identify which source these metrics are for because each sub-pipeline is unique to each source.
```

Comment:
```suggestion
Metrics in source coordination are interpreted differently depending on which source is configured. The format of source coordination metrics is `<sub-pipeline-name>_source_coordinator_<metric-name>`. You can use the sub-pipeline name is used to identify which source these metrics are for because each sub-pipeline is unique to each source.
```